### ISSUE TITLE
About the svhn dataset

### ISSUE THREAD

**Issue by whobefore** [mingyuliutw/UNIT#4]
Thanks for your source code for domain adaptation! However, I found the training set used in SVHN-MNIST experiment, you choose the 'extra' split of SVHN but many of other paper's settings are the 'train' split with 73257 images. And I change the split into 'train', the training process is much slower than before. It's accuracy on MNIST test set is 0.3773 in 58k iteration. Have you tried the 'train' split? Will it take more time to train? Thanks a lot!

**Comment by mingyuliutw** [comment]
The baseline in my case was the DTN work (https://arxiv.org/pdf/1611.02200.pdf), which used the extra set, containing 531,131 images. I have never tested using the train set. I would guess you need to reduce the network capacity to make it work because the number of images is smaller. .
